src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    public double getArea();

```

Comment:
There is no need to declare interface methods as `public`, as they are public by default. You can safely remove the `public` modifier.